ui/common/build.gradle.kts

```diff
@@ -45,6 +48,9 @@ kotlin {
                 implementation(compose.material)
                 api(libs.com.arkivanov.decompose)
                 api(libs.com.arkivanov.decompose.extensions.compose.jetbrains)
+                implementation("com.alialbaali.kamel:kamel-image:0.4.0")
+                implementation("it.skrape:skrapeit:1.2.2")
```

Comment:
Is this also JVM only? Maybe we can use this as a resource and roll our own purely kotlin parser which also supports iOS

Reply:
yup yup jvm only. I think this gets us off the ground and agree we would need something more robust in future